.github/workflows/ci.yml

```diff
@@ -75,3 +75,39 @@ jobs:
           path: |
             target/scoverage-report/**/*
             target/test-reports/**/*
+
+  docker:
+    runs-on: ubuntu-latest
+    needs: sbt-build-test
+    if: ${{ github.ref == 'refs/heads/pg-store' }}
+
+    steps:
+      - name: checkout sources
+        uses: actions/checkout@v2
+
+      - name: Fetch jar
+        uses: actions/download-artifact@v2
+        with:
+          name: rpki-publication-server-jar
+
+      - name: Login to Docker Hub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_HUB_USERNAME }}
+          password: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
+
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Build and push
+        id: docker_build_publication_server
+        uses: docker/build-push-action@v2
+        with:
+          context: ./
+          file: ./Dockerfile
+          push: true
+          tags: ripencc/rpki-publication-server:latest
```

Comment:
Can we add the `labels` param for the commit here?

Reply:
Done (hopefully, but I'll check the result from the workflow :D)